src/styles/main.scss

```diff
@@ -0,0 +1,76 @@
+$grey: #eee;
+$black: #000;
+$pink: #ffbfcb;
+$day-size: 100px;
+$border-size: 1px;
+$font-size: 30px;
+$gap-size: 1px;
+$padding-size: 10px;
+$animation-duration: 0.5s;
+
+// Основные стили для календаря
+body,
+html {
+  height: 100%;
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
```

Comment:
According to BEM rules, each Block should be in a separate file
Same for variables

Also, no need to leave comments if the code is self-descriptive enough 🙂 But even if you left comments they should be in English

Generally, everything else looks good - approved ✔️

Reply:
So in that case here should be 3 blocks: index, calendar and variables, right?